content/docs/04-clusters/03-edge/09-site-deployment/05-site-installation/04-site-user-data.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Apply Site User Data"
+metaTitle: "Site User Data"
+metaDescription: "Learn how to create a secondary Edge Installer configuration user data."
```

Comment:
```suggestion
metaDescription: "Learn how to create a secondary Edge Installer user data configuration."
```

Reply:
I might have these terms reversed here. I thought from my previous review that user data was a configuration file. I see in the Overview section (line 16) that it's 'configuration user data'.